dev-util/pkgconf/pkgconf-1.3.90.recipe

```diff
@@ -0,0 +1,121 @@
+SUMMARY="A lightweight pkg-config replacement"
+DESCRIPTION="A program which helps to configure compiler and linker flags for \
+development libraries. It is similar to pkg-config from freedesktop.org."
+HOMEPAGE="http://pkgconf.org/"
+COPYRIGHT="2011-2017 pkgconf authors"
+LICENSE="ISC"
+REVISION="1"
+SOURCE_URI="https://distfiles.dereferenced.org/pkgconf/pkgconf-$portVersion.tar.xz"
+CHECKSUM_SHA256="112db7f957816901b5fe448715c0a9e38f171d4794c44075b0734a0e7fcfba36"
+PATCHES="pkgconf-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix="$secondaryArchSuffix"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=""
+fi
+
+libVersion="2.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	pkgconf$secondaryArchSuffix = $portVersion
+	cmd:pkgconf$secondaryArchSuffix = $portVersion
+	lib:libpkgconf$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	pkgconf${secondaryArchSuffix}_devel = $portVersion
+	devel:libpkgconf$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	pkgconf$secondaryArchSuffix == $portVersion base
+	"
+
+SUMMARY_pkgconfig="$SUMMARY (pkg-config compat symlink)"
+DESCRIPTION_pkgconfig="$DESCRIPTION (pkg-config compatibility symlink)"
+
+PROVIDES_pkgconfig="
+	pkgconf${secondaryArchSuffix}_pkgconfig = $portVersion
+	cmd:pkg_config$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_pkgconfig="
+	pkgconf$secondaryArchSuffix == $portVersion base
+	"
+CONFLICTS_pkgconfig="
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:cmp
+	cmd:diff
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:make
+	cmd:sed
+	"
+
+TEST_REQUIRES="
+	devel:libatf_c$secondaryArchSuffix
+	"
+TEST_PREREQUIRES="
+	cmd:kyua$commandSuffix
+	"
+BUILD_REQUIRES+="$TEST_REQUIRES"
+BUILD_PREREQUIRES+="$TEST_PREREQUIRES"
+
+defineDebugInfoPackage pkgconf$secondaryArchSuffix \
+	"$binDir/pkgconf" \
+	"$libDir/libpkgconf.so.$libVersion"
+
+BUILD()
+{
+	local pkgConfSubDir="lib/${secondaryArchSuffix/_}/pkgconfig"
```

Comment:
```diff
-	local pkgConfSubDir="lib/${secondaryArchSuffix/_}/pkgconfig"
+	local pkgConfSubDir="$lrelativeLibDir/pkgconfig"
```
This change is neutral on any secondary arch (e.g. x86), but not primary archs (e.g. x86_64).

Reply:
It's not really an error, `//` :arrow_right: `/`. But yours looks way better, so I'm going to use it instead.